chainio/txmgr/geometric/geometric_example_test.go

```diff
@@ -23,6 +24,37 @@ var (
 	chainid = big.NewInt(31337)
 )
 
+func createTx(client eth.HttpBackend, address common.Address) (*types.Transaction, error) {
+	zeroAddr := common.HexToAddress("0x0")
+	nonce, err := client.PendingNonceAt(context.TODO(), address)
+	if err != nil {
+		return nil, utils.WrapError("Failed to get PendingNonceAt", err)
+	}
+	return types.NewTx(&types.DynamicFeeTx{
+		To:    &zeroAddr,
+		Nonce: nonce,
+	}), nil
+}
+
+func createTxMgr(rpcUrl string, ecdsaPrivateKey *ecdsa.PrivateKey) (eth.HttpBackend, *GeometricTxManager, error) {
+	logger := logging.NewTextSLogger(os.Stdout, &logging.SLoggerOptions{})
+	client, err := ethclient.Dial(rpcUrl)
+	if err != nil {
+		return nil, nil, err
+	}
+	signerV2, signerAddr, err := signerv2.SignerFromConfig(signerv2.Config{PrivateKey: ecdsaPrivateKey}, chainid)
+	if err != nil {
+		return nil, nil, err
+	}
+	wallet, err := wallet.NewPrivateKeyWallet(client, signerV2, signerAddr, logger)
+	if err != nil {
+		return nil, nil, err
+	}
+	reg := prometheus.NewRegistry()
+	metrics := NewMetrics(reg, "example", logger)
+	return client, NewGeometricTxnManager(client, wallet, logger, metrics, GeometricTxnManagerParams{}), nil
+}
+
 func ExampleGeometricTxManager() {
 	anvilC, err := testutils.StartAnvilContainer("")
 	if err != nil {
```

Comment:
I think we should also avoid the use of panics in the examples as well

Reply:
Done